wtr-utils.js

```diff
@@ -73,12 +54,17 @@ const getChangedPackages = () => {
 };
 
 /**
- * Get all available packages.
+ * Get all available packages with unit tests.
  */
-const getAllPackages = () => {
+const getAllUnitPackages = () => {
   return fs
     .readdirSync('packages')
-    .filter((dir) => fs.statSync(`packages/${dir}`).isDirectory() && fs.existsSync(`packages/${dir}/test`));
+    .filter(
+      (dir) =>
+        fs.statSync(`packages/${dir}`).isDirectory() &&
+        fs.existsSync(`packages/${dir}/test`) &&
+        glob.sync(`packages/${dir}/test/*.test.js`).length > 0
```

Comment:
I wonder if it might be reasonable to extend the search: `packages/${dir}/test/**/*.test.js`.

Reply:
Didn't find any tests in sub-folders, however. So we can skip it.

Reply:
This is intentional. We use subfolders for visual tests and typings. Ideally we would have also `test/unit` folders.